src/DropzoneTrait.php

```
@@ -0,0 +1,171 @@
+
+    $files_path = $this->getMinkParameter('files_path');
+
+    if (empty($files_path)) {
+      throw new \RuntimeException('The Mink "files_path" parameter is not configured.');
+    }
+
+    $base = rtrim((string) realpath((string) $files_path), DIRECTORY_SEPARATOR);
+    $full_path = $base . DIRECTORY_SEPARATOR . ltrim($path, '/');
+
+    if (!is_file($full_path)) {
+      throw new \RuntimeException(sprintf('The fixture file "%s" does not exist.', $full_path));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '160', column '17'). (undefined)

(MissingImport)